go.mod

```diff
@@ -162,3 +162,5 @@ require (
 
 // TODO: https://github.com/knative/serving/issues/14597
 replace github.com/gorilla/websocket => github.com/gorilla/websocket v1.5.0
+
+replace knative.dev/networking v0.0.0-20231219131858-8c7897c8b9a0 => github.com/KauzClay/networking v0.0.0-20240103183713-99c292ac8072
```

Comment:
Can the changes you need from `github.com/KauzClay/networking` go to `knative.dev/networking`?

Reply:
yeah I actually have another PR open with them here: https://github.com/knative/networking/pull/904

Reply:
this was just a way to get the PR tests to run with those changes, and see if I was breaking anything major in serving